service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/generator/C43ChildArrangementOrderDocumentParameterGenerator.java

```diff
@@ -100,22 +105,29 @@ public DocmosisTemplates template() {
     private String buildOrderDetails(ManageOrdersEventData eventData) {
         StringBuilder stringBuilder = new StringBuilder();
 
-        stringBuilder.append("The Court orders");
+        stringBuilder.append("The Court orders,");
```

Comment:
Could we get away with just doing StringBuilder stringBuilder = new StringBuilder("The Court orders,"); here?